Error 2021-09-16 17:43:16: Failed to check if file needs to be fixed [EntryNotFound (FileSystemError): Unable to read file '/root/.config/Code/User/settings.json' (Error: Unable to resolve non-existing file '/root/.config/Code/User/settings.json')
	at _handleError (/snap/code/74/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:160087)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async y.readText (/root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:9:314351)
	at async p.doesFileNeedToBeFixed (/root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:59:823969)
	at async /root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:59:823096
	at async Promise.all (index 0)
	at async p.getFilesToBeFixed (/root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:59:823042)
	at async p.updateTestSettings (/root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:59:822669)] {
  code: 'FileNotFound'
}
Error 2021-09-16 17:43:16: Failed to check if file needs to be fixed [EntryNotFound (FileSystemError): Unable to read file '/home/hp-precily/shubham/dailywork/alp/.vscode/settings.json' (Error: Unable to resolve non-existing file '/home/hp-precily/shubham/dailywork/alp/.vscode/settings.json')
	at _handleError (/snap/code/74/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:160087)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async y.readText (/root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:9:314351)
	at async p.doesFileNeedToBeFixed (/root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:59:823969)
	at async /root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:59:823096
	at async Promise.all (index 1)
	at async p.getFilesToBeFixed (/root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:59:823042)
	at async p.updateTestSettings (/root/.vscode/extensions/ms-python.python-2021.9.1230869389/out/client/extension.js:59:822669)] {
  code: 'FileNotFound'
}
Python interpreter path: /usr/bin/python3.8
Starting Pylance language server.
> conda --version
Error 2021-09-16 18:23:59: Error while running worker[Worker 1]. undefined
Error 2021-09-16 18:23:59: Error while running worker[Worker 2]. undefined
